eng/Packages.Data.props

```diff
@@ -182,10 +182,16 @@
 	
     <PackageReference Update="Microsoft.Azure.Devices.Client" Version="1.27.0" />
 	
-	 <!-- Mgmt sdk packages-->
+    <!-- Mgmt sdk packages-->
     <PackageReference Update="Azure.ResourceManager.Resources" Version="1.0.0-preview.2" />
     <PackageReference Update="Azure.ResourceManager.Compute" Version="1.0.0-preview.2" />
     <PackageReference Update="Azure.ResourceManager.Network" Version="1.0.0-preview.2" />
     <PackageReference Update="Azure.ResourceManager.Storage" Version="1.0.0-preview.2" />
   </ItemGroup>
+
+  <!-- Perf only packages -->
+  <ItemGroup Condition="'$(IsPerfProject)' == 'true'">
```

Comment:
I assume you plan to handle the merge conflicts between this and the Stress PR.

Reply:
Yep I knew there would be merge conflicts but wanted to split these into two PRs.